schemas/isd.json

```diff
@@ -0,0 +1,236 @@
+          "description": "(X, Y, Z) location of sensor in body-fixed frame.",
+          "type": "array",
+          "items": {
+             "type": "array",
+             "items" : {
+                "type": "number",
+                "minItems": 3,
+                "maxItems": 3
+            }
+         }
+      },
+      "sensor_velocity" : {
+         "description": "(v_x, v_y, v_z) velocity of sensor in body-fixed frame.",
+         "type": "array",
+         "items": {
+            "type": "array",
+            "items" : {
+               "type": "number",
+               "minItems": 3,
+               "maxItems": 3
+            }
+         }
+      },
+      "sun_position" : {
+         "description": "(x, y,z) of the sun in body-fixed frame.",
+         "type": "array",
+         "items": {
+            "type": "array",
+            "items" : {
+               "type": "number",
+               "minItems": 3,
+               "maxItems": 3
+            }
+         }
+      },
+      "sun_velocity" : {
+         "description": "(v_x, v_y, v_z) velocity of sun in body-fixed frame.",
+         "type": "array",
+         "items": {
+            "type": "array",
+            "items" : {
+               "type": "number",
+               "minItems": 3,
+               "maxItems": 3
+            }
+         }
+      },
+      "sensor_orientation" : {
+         "description": "Orientation of the sensor provided in quaternions",
+         "type": "array",
+         "items": {
+            "type": "array",
+            "description": "Each quaternion is of the form: (scalar, vector, vector, vector)" 
+            "items" : {
+               "type": "number",
+               "minItems": 4 ,
+               "maxItems": 4
+            }
+         }
+      },
+      "line_scan_rate": {
+         "description": "The line rates of the detector map with the associated start line and time",
+         "type": "array",
+         "items": {
+            "type": "array",
+             "items" : {
+               "type": "number",
+               "default: "-1",
+               "description": "The (start_line, line_start_time, exposure_time). The exposure time is
+                               in seconds and any non-provided values will be set to -1."
+               "minItems": 3,
+               "maxItems": 3
+            }
+         }
+      }, 
+      "detector_sample_summing": {
+         "description": "The summing in the sample-direction. A summing of 1 indicates no summing, 
+                         summing of 2 indicates 2 pixel summing, 4 indicates 4 pixel summing, etc",
+         "type": "number"
+      },
+      "detector_line_summing": {
+         "description": "The summing in the line-direction",
+         "type": "number"
+      },
+      "apply_light_time_correction": {
+         "description": "A flag to indcate whether the light-time correction should be applied",
+         "type": "bool"
+      },
+      "number_of_ephem": {
```

Comment:
The spec is mixing 'quat' and 'quaternion' and 'ephem' and 'ephemeris'.  I would rather just use the full word to be very explicit, but consistency is more important than preference.

Reply:
:+1:

Reply:
I updated this to use the full words ephemeris (or ephemerides) and quaternion.